homeassistant/components/tibber/sensor.py

```diff
@@ -201,36 +213,36 @@ async def _async_callback(self, payload):
         self.async_write_ha_state()
 
     @property
-    def available(self):
+    def available(self) -> Any:
         """Return True if entity is available."""
         return self._tibber_home.rt_subscription_running
 
     @property
-    def model(self):
+    def model(self) -> str:
         """Return the model of the sensor."""
         return "Tibber Pulse"
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the sensor."""
         return f"Real time consumption {self._name}"
 
     @property
-    def should_poll(self):
+    def should_poll(self) -> bool:
         """Return the polling state."""
         return False
 
     @property
-    def unit_of_measurement(self):
+    def unit_of_measurement(self) -> str:
         """Return the unit of measurement of this entity."""
         return POWER_WATT
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return a unique ID."""
         return f"{self.device_id}_rt_consumption"
 
     @property
-    def device_class(self):
+    def device_class(self) -> Any:
```

Comment:
```suggestion
    def device_class(self) -> str:
```

Reply:
Gives:
> homeassistant/components/tibber/sensor.py:248: error: Returning Any from function declared to return "str"  [no-any-return]

Reply:
I think you need to import `DEVICE_CLASS_POWER` from `homeassistant.const`.